extensions/3DTILES_metadata/examples/terrainpack/terrainpack.json

```diff
@@ -0,0 +1,375 @@
+              "wff_version": "1.0",
+              "runtime_enabled": true,
+              "geometry_model": "3DOBJECTS",
+              "content_type": "VECTOR",
+              "capture_platform": "UNKNOWN",
+              "capture_sensor": "UNKNOWN",
+              "provider": "Maxar",
+              "id": "2e7444d7-e816-48c6-8786-c24ab354422b",
+              "created": "2020-10-05T13:46:46.672632Z",
+              "semantic": "_WFF_DATASET"
+            }
+          },
+          "3D_Buildings": {
+            "name": "3D_Buildings",
+            "class": "_WFF_DATASET",
+            "properties": {
+              "wff_version": "1.0",
+              "runtime_enabled": true,
+              "geometry_model": "3DOBJECTS",
+              "content_type": "MESH",
+              "capture_platform": "SATELLITE",
+              "capture_sensor": "ELECTRO_OPTICAL",
+              "provider": "Maxar",
+              "id": "2e7444d7-e816-48c6-8786-c24ab3544244",
+              "created": "2020-10-05T13:46:46.672632Z",
+              "semantic": "_WFF_DATASET"
+            }
+          },
+          "Bridges": {
+            "name": "Bridges",
+            "class": "_WFF_DATASET",
+            "properties": {
+              "wff_version": "1.0",
+              "runtime_enabled": true,
+              "geometry_model": "3DOBJECTS",
+              "content_type": "MESH",
+              "capture_platform": "SATELLITE",
+              "capture_sensor": "ELECTRO_OPTICAL",
+              "provider": "Maxar",
+              "id": "2e7444d7-e816-48c6-8786-c24ab3544244",
+              "created": "2020-10-05T13:46:46.672632Z",
+              "semantic": "_WFF_DATASET"
+            }
+          },
+          "Vertical_Obstructions": {
+            "name": "Vertical Obstructions",
+            "class": "_WFF_DATASET",
+            "properties": {
+              "wff_version": "1.0",
+              "runtime_enabled": true,
+              "geometry_model": "3DOBJECTS",
+              "content_type": "MESH",
+              "capture_platform": "MIXED",
+              "capture_sensor": "UNKNOWN",
+              "provider": "SE Core",
+              "id": "2e7444d7-e816-48c6-8786-c24ab3544245",
+              "created": "2020-10-05T13:46:46.672632Z",
+              "semantic": "_WFF_DATASET"
+            }
+          },
+          "Airport_Features": {
+            "name": "Airport Features",
+            "class": "_WFF_DATASET",
+            "properties": {
+              "wff_version": "1.0",
+              "runtime_enabled": true,
+              "geometry_model": "3DOBJECTS",
+              "content_type": "MESH",
+              "capture_platform": "MIXED",
+              "capture_sensor": "UNKNOWN",
+              "provider": "SE Core",
+              "id": "2e7444d7-e816-48c6-8786-c24ab3544246",
+              "created": "2020-10-05T13:46:46.672632Z",
+              "semantic": "_WFF_DATASET"
+            }
+          }
+        }
+      },
+      "namespaces": {
```

Comment:
To make this example easier for a newcomer to understand, could we move the `namespaces` key before the `metadata` key? This way the definitions come before the instances.